op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -0,0 +1,147 @@
+package proofs
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	actionsHelpers "github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/proofs/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+const OperatorFeeScalar = uint32(20000)
+const OperatorFeeConstant = uint64(500)
+
+func Test_Operator_Fee_Constistency(gt *testing.T) {
```

Comment:
Question for reviewers: Is this action test sufficient for the operator fee feature? Should other action tests covering operator fee be added here or in a follow-on PR?

Reply:
Were these tests written yet?

Reply:
No. Tracking here as follow up work: https://github.com/ethereum-optimism/optimism/issues/14626